Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:328.0.0
```

Comment:
It doesn't look like we can upgrade the base docker image - see previous PR - https://github.com/nytimes/drone-gae/pull/104

Reply:
@tonglil said we may be able to call it a new major version, though?